reminder/module.py

```diff
@@ -0,0 +1,465 @@
+    ):
+        text = await self._process_text(ctx, datetime_str, text)
+
+        try:
+            date = utils.time.parse_datetime(datetime_str)
+        except dateutil.parser.ParserError:
+            await ctx.reply(
+                _(
+                    ctx,
+                    "I don't know how to parse `{datetime_str}`, please try again.",
+                ).format(datetime_str=datetime_str)
+            )
+            return
+
+        ReminderItem.add(
+            guild_id=ctx.guild.id,
+            author_id=ctx.author.id,
+            remind_id=ctx.author.id,
+            permalink=ctx.message.jump_url,
+            message=text,
+            origin_date=ctx.message.created_at,
+            remind_date=date,
+        )
+
+        date = date.strftime("%d.%m.%Y %H:%M")
+
+        await bot_log.debug(
+            ctx.author,
+            ctx.channel,
+            f"Reminder created for {ctx.author.name}",
+        )
+
+        await ctx.message.add_reaction("✅")
+        await ctx.message.author.send(
+            _(ctx, "Reminder for you created. Reminder will be sent: {date}").format(
+                date=date
+            )
+        )
+
+    @commands.guild_only()
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.check(check.acl)
+    @commands.command()
+    async def remind(self, ctx, member: nextcord.Member, datetime_str: str, text: str):
+        text = await self._process_text(ctx, datetime_str, text)
+
+        try:
+            date = utils.time.parse_datetime(datetime_str)
+        except dateutil.parser.ParserError:
+            await ctx.reply(
+                _(
+                    ctx,
+                    "I don't know how to parse `{datetime_str}`, please try again.",
+                ).format(datetime_str=datetime_str)
+            )
+            return
+
+        ReminderItem.add(
+            guild_id=ctx.guild.id,
+            author_id=ctx.author.id,
+            remind_id=member.id,
+            permalink=ctx.message.jump_url,
+            message=text,
+            origin_date=ctx.message.created_at,
+            remind_date=date,
+        )
+
+        date = date.strftime("%d.%m.%Y %H:%M")
+
+        await bot_log.debug(
+            ctx.author,
+            ctx.channel,
+            f"Reminder created for {member.name}",
+        )
+
+        await ctx.message.add_reaction("✅")
+        await ctx.message.author.send(
+            _(ctx, "Reminder for {name} created. Reminder will be sent: {date}").format(
+                name=member.display_name, date=date
+            )
+        )
+
+    @commands.guild_only()
+    @commands.check(check.acl)
+    @commands.group(name="reminders")
+    async def reminders_(self, ctx):
+        await utils.discord.send_help(ctx)
+
+    @commands.guild_only()
+    @commands.check(check.acl)
+    @reminders_.command(name="get")
+    async def reminders_get(self, ctx, status: str = "WAITING"):
```

Comment:
this could be a `reminder list` command. Also not guild_only

Reply:
Renamed. Guild lock depends on discussion with sinus-x.